MRG: Project 2 Path Report

jcpalumbo3 opened the pull request:
Editing the final report documentation for the path subproject. Let me know if there's anything you'd like to include. 
@sicihuang @alannaiverson @jJasonWang @aksam-ahmad @togawa28

sicihuang commented on doc/source/report/path.rst:
```diff
@@ -75,81 +57,78 @@ As we begin our analysis, we would like to explore the following points:
+
+**Step 3**: Choose Key Features
+
+The path features we wrapped up into functions are as follows: 
```
This line was italicized for some reason. Maybe you want to check rst file style?

sicihuang commented on doc/source/report/path.rst:
```diff
@@ -75,81 +57,78 @@ As we begin our analysis, we would like to explore the following points:
+Results
+-------------------------
+
+.. figure:: figure/dist_path.png
```
We should comment on these plots. For example:
"Strain 0 appeared to be the most active overall. For all mice in this strain, activeness peaked around days 6-8 and fell after that. In contrast, there is a huge variation in the activeness among mice of strain 1."

aksam-ahmad commented:
**Further Studies**

In the future, we would like to explore the following areas:

- Further data smoothing: We would like to further develop a robust methodology to detect points which could be deemed ‘measurement noise’. Measurement noise can be defined as consecutive observations which indicate high-turning angles (i.e., above 150 degrees) over very short time intervals. Such observations would skew results on number of sharp turns, average turning angles, etc. It is possible the measurement noise obscures the path behavior differences between strains.

- Path behavior as it relates to active state lifestyle: We would like to explore the influence of a mouse’s active state behaviors on its path behavior. A possible question we may be answer is the following: how does hunger and thirst affect the mouse’s path diversity? (i.e., does the mouse get sluggish after having eaten and therefore takes shorter, slower paths?)

- Obtaining optimal tuning parameters: We would like to determine which parameters would be optimal for analysis purposes. These parameters include, but are not limited to, the cutoff time for a path, the minimum path length (in observations) and turning angle threshold for noise detection. 

- Use path features for classification: We would like to use the features calculated within the functions and apply machine learning techniques to determine whether path features (speed, turning angles, distance traveled, etc.) can be used for classification.

jcpalumbo3 commented:
I added @aksam-ahmad 's future studies section, updated the results, and added additional remarks. I think after adding @togawa28 results, we will just one final check.

jcpalumbo3 commented:
@jarrodmillman This is ready to merge. Coverage went down slightly, but the issue doesn't seem to be related to the rst file as far as we can tell.

jarrodmillman commented:
@jcpalumbo3 Thanks, I will take it from here.

jarrodmillman commented:
Rebased in #204
